config/routes.rb

```diff
@@ -10,7 +10,7 @@
       end
     end
     resources :checkouts, concerns: :legacy_return_routes
-    resources :orders, concerns: :legacy_return_routes
+    resources :orders, except: [:show], concerns: :legacy_return_routes
```

Comment:
Reading [this](http://guides.rubyonrails.org/routing.html#routing-concerns), looks like we should try and do this with a `namespace` rather than `resources` to make sure it doesn't clobber anything.